packages/experimental/src/experimental-list/task-item.tsx

```diff
@@ -114,9 +123,14 @@ export const TaskItem: React.FC< TaskItemProps > = ( {
 	actionLabel,
 	...listItemProps
 } ) => {
+	const [ isTaskExpanded, setTaskExpanded ] = useState( expanded );
+	useEffect( () => {
+		setTaskExpanded( ! isTaskExpanded );
```

Comment:
Could we set it to `expanded` instead? This could cause some weird state inconsistencies if we only toggle on `expanded` change.

Reply:
> Are you also able to rebase with main, I keep seeing the old task list (the one with the descriptions) and it freaks me out. Your changes shouldn't affect this, but I want to make sure.

In relation to the above, I think this change actually causes this. `useEffect` is also called on `init`, so it already toggles the expanded boolean on first render, causing everything that is set to be collapsed to be expanded instead.

Reply:
Good idea! This was fixed in the commit 447097e